lib/broccoli/ember-app.js

```diff
@@ -1394,7 +1394,15 @@ EmberApp.prototype.testFiles = function(coreTestTree) {
     overwrite: true,
     annotation: 'TreeMerger (addon-test-support)'
   });
-  var transpiledAddonTestSupportTree = new Babel(addonTestSupportTree, this._prunedBabelOptions());
+
+  var transpiledAddonTestSupportTree
+  if (this._addonInstalled('ember-cli-babel')) {
```

Comment:
We want to call `preprocessJs` no matter what addons somebody has installed, so it was correct in `lib/models/addon`. We just need to have the back of people relying on the (previously) default babel transpile.

```javascript
var transpiledAddonTestSupportTree
if (!this._addonInstalled('ember-cli-babel') && "we didn't tell it to stop double transpiling") {
  if ("addonTestSupportTree has files") {
    this.ui.writeDeprecateLine('Must include ember-cli-babel if you want to transform the tree.');
  }
  transpiledAddonTestSupportTree = new Babel(addonTestSupportTree, this._prunedBabelOptions());
}
```